packages/cli/src/helpers.ts

```diff
@@ -0,0 +1,98 @@
+import os from 'os';
+import { exec, spawnSync } from 'child_process';
+import prompts from 'prompts';
+import { magentaBright } from 'chalk';
+
+export async function setupAnvil(): Promise<void> {
+  const needsCargo = !(await hasCargo());
+  if (needsCargo) {
+    const cargoResponse = await prompts({
+      type: 'confirm',
+      name: 'confirmation',
+      message: 'Cannon requires Cargo (https://doc.rust-lang.org/cargo/). Install it now?',
```

Comment:
cargo is probably actually not needed now that we have been upstreamed (cargo is a build tool used by rust, but you dont need to build anything). At least, its not needed by the synthetix cli anymore